### ISSUE TITLE
Improve negate plugin helptext

### ISSUE THREAD

**Issue by euniceremoquillo** [monitoring-plugins/monitoring-plugins#1986]
The negate plugin's helptext is currently ambiguous as it says that it will negate the status of a plugin:
```
Negates the status of a plugin (returns OK for CRITICAL and vice-versa).
Additional switches can be used to control which state becomes what.
```
Example re-work of the helptext could be:
```
Negates only the return code of a plugin (returns OK for CRITICAL and vice-versa) by default.
Additional switches can be used to control:
- which state becomes what
- changing the plugin output text to match the return code
```

**Comment by RincewindsHat** [comment]
Hi @euniceremoquillo ,
The help could indeed be more clear. Would you like to make an suggestion as a PR? I think it would be better if someone "outside" of the project would do this.

**Comment by euniceremoquillo** [comment]
https://github.com/monitoring-plugins/monitoring-plugins/pull/1987

**Comment by RincewindsHat** [comment]
fixed by #1987
